tools/sizing/size_finder_test.go

```diff
@@ -0,0 +1,176 @@
+package sizing
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/Layr-Labs/eigenda/api/clients/v2"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/coretypes"
+	"github.com/docker/go-units"
+	"github.com/stretchr/testify/require"
+)
+
+const minBlobSize = uint64(128 * units.KiB)
+const maxBlobSize = uint64(16 * units.MiB)
+
+var defaultPayloadForm = clients.GetDefaultPayloadClientConfig().PayloadPolynomialForm
+
+// Derive the real size of a blob for a given payload by creating a payload and converting it to a blob.
+func deriveRealBlobSize(t *testing.T, payloadSize uint64) uint64 {
+
+	rawBytes := make([]byte, payloadSize)
+	payload := coretypes.NewPayload(rawBytes)
+	blob, err := payload.ToBlob(defaultPayloadForm)
+	require.NoError(t, err)
+
+	return uint64(blob.BlobLengthBytes())
+}
+
+// This function generates a table containing optimum blob sizes. It is intended to be run manually.
+func TestGenerateOptimumSizeTable(t *testing.T) {
+
+	// Uncomment this to generate an optimum size table.
+	t.Skip() // Do not merge with this line uncommented.
```

Comment:
> // Do not merge with this line uncommented.

It's uncommented. I think the docs here need to be flipped around

Reply:
fixed